.github/workflows/codeql.yml

```diff
@@ -1,11 +1,11 @@
----
 # For most projects, this workflow file will not need changing; you simply need
 # to commit it to your repository.
 #
 # You may wish to alter this file to override the set of languages analyzed,
 # or to provide custom queries or build logic.
-name: "CodeQL"
 
+---
+name: "CodeQL"
```

Comment:
How is this change related to error `every step must define a uses or run key`? Or is this for yamllint?
Strange thing is that I do not see CodeQL action running for this PR.
But it seems that the patch from Shah fixed the issue and CodeQL action is green again.